Sources/CardPayments/Models/CardRequest.swift

```diff
@@ -11,14 +11,24 @@ public struct CardRequest {
     /// 3DS authentication launch option
     public let sca: SCA
     
+    /// The option to vault with purchase
+    public let shouldVault: Bool
```

Comment:
We should update this PR to match the [implementation decided on Android](https://github.com/paypal/Android-SDK/pull/157#discussion_r1263037902). So instead of 2 separate properties, we want a [Vault object](https://github.com/paypal/Android-SDK/blob/main/CardPayments/src/main/java/com/paypal/android/cardpayments/Vault.kt) with an optional `customerID`.

Reply:
Ope, I think you already made this change and my GH was out of date for some reason.